drivers/SmartThings/matter-switch/fingerprints.yml

```
@@ -1,3 +1,26 @@
+matterManufacturer:
+#Eve
+  - id: "Eve/Energy/US"
+    deviceLabel: Eve Energy US
+    vendorId: 0x130a
```

Just a nit, but good to be consistent across the fingerprints. Two are currently lowercase and two are currently uppercase. I prefer uppercase personally.

```suggestion
    vendorId: 0x130A
```